lib/atlas/dataset/derived_dataset.rb

```diff
@@ -4,7 +4,7 @@ class Dataset::DerivedDataset < Dataset
 
     attribute :base_dataset, String
     attribute :scaling,      Preset::Scaling
-    attribute :init,         Hash[Symbol => Float]
+    attribute :init,         Hash[InitializerInput => Float]
```

Comment:
This is inconsistent with `Preset` which just uses `{Symbol => Float}` for user values. That might be fine, but:

* Does the document load if a non-existent initializer input is specified?
* Does this prevent us from adding [a validation which asserts the existence of each input](https://github.com/quintel/atlas/blob/master/lib/atlas/preset.rb#L32-L47)? (That's important so that mistakes made when manually editing documents can be caught by the ETSource specs).